app/services/onlinetesting/OnlineTestExpiryService.scala

```diff
@@ -65,7 +66,7 @@ class OnlineTestExpiryServiceImpl(
     }
 
   override def commitExpiredProgressStatus(expiringTest: ExpiringOnlineTest): Future[Unit] =
```

Comment:
If this is for phase1, maybe we should include that in the name of the method, i.e. commitPhase1TestsExpiredProgressStatus